### PR TITLE
Update 2201.5.0 release note with OpenAPI changes

### PR DESCRIPTION
**Pull request by aneeshafedo**
Release notes related to the fix https://github.com/ballerina-platform/openapi-tools/issues/1211

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/openapi.md [hunk 1]
```diff
@@ -23,23 +23,26 @@ bal openapi [-i | --input] <openapi-contract-file-path>
...
+| `--operations`        | To generate the Ballerina client or service stub with a subset of operations defined in the OpenAPI contract, use the `--operations` option and specify the operations you need as specified in the OpenAPI definition.<br><br>**E.g.,** `bal openapi -i <openapi-contract> [--operations <"op1", "op2">]`                                                                                                                                                                                                                       |
+| `--license`           | To generate the Ballerina files with the given copyright or license header, you can use this `--license` flag with your copyright text.<br><br>**E.g.,** `bal openapi -i <openapi-contract> [--license <license-file-path>]`                                                                                                                                                                                                                                                                                                     |
```
Comment (NipunaRanasinghe) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/openapi.md:39]: Can you please re-check the rendering with the these changes to the CLI reference? github rich diff shows a weird rendering the new changes made on the table

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 2]
```diff
@@ -149,6 +149,13 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 #### GraphQL Tool
 
 #### OpenAPI Tool
+Added `--with-service-type` command option to generate a ballerina service with a service-type object. This is useful in validating the service against the OpenAPI contract given.
```
Comment (NipunaRanasinghe) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:152]:
```suggestion
- Added `--with-service-type` command option to generate a ballerina service with a service-type object. This is useful in validating the service implementation against the OpenAPI contract given.
```

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/openapi.md (hunk 1 above)
Comment (aneeshafedo) [reply swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/openapi.md]: Yes, thank you. There was an unintended formatting change. Updated the PR.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md (hunk 2 above)
Comment (aneeshafedo) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md]: Updated

**Pull request closed without merging**